index.bs

```diff
@@ -129,25 +134,65 @@ The {{Accelerometer/z!!attribute}} attribute of the {{Accelerometer}}
 interface represents the <a>acceleration</a> along Z-axis.
 This attribute returns [=map/value=] for [=latest reading=]["z"] [=map/entry=].
 
-### Accelerometer.includesGravity ### {#accelerometer-includesGravity}
+The LinearAccelerometer Interface {#linear-accelerometer-interface}
```

Comment:
Do people actually call it linear accelerometer... I dont think that is a real term. We probably should use LinearAccelerationSensor instead